.travis.yml

```diff
@@ -48,9 +48,11 @@ install:
     fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$CXX" = "g++" ]; then export CXX="g++-6"; fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ] && [ "$CXX" == "clang++" ]; then export CXX="clang++-3.8"; fi
+  - pip install -r travis/python_reqs.txt
 
 script:
-  - bash .travis.sh
+  - bash travis/check_style.sh
+  - bash travis/travis_main.sh
```

Comment:
Ah! You already done it, sorry! :-) The search didn't work in the GH UI due to the very large diff... Forget the last comment!